roles/kustomize_deploy/tasks/install_operators.yml

```diff
@@ -260,3 +260,27 @@
           type: Available
           status: "True"
         wait_timeout: 300
+
+- name: Wait until OpenStack operators are deployed and ready
+  when: not cifmw_kustomize_deploy_generate_crs_only
+  kubernetes.core.k8s_info:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
```

Comment:
In the ci-framework the calls to `kubernetes.core.k8s)_*` that I have seen also include these lines:

```yaml
    api_key: "{{ cifmw_openshift_token | default(omit) }}"
    context: "{{ cifmw_openshift_context | default(omit) }}"
```